nav2_planner/src/planner_server.cpp

```diff
@@ -52,6 +53,12 @@ PlannerServer::PlannerServer(const rclcpp::NodeOptions & options)
   declare_parameter("planner_plugins", default_ids_);
   declare_parameter("expected_planner_frequency", 1.0);
 
+  nav2_util::declare_parameter_if_not_declared(
+    this, "z_score", rclcpp::ParameterValue(2.55));
+
+
+  get_parameter("z_score", z_score_);
```

Comment:
In the parameter / variable name, it would be nice to have some more context to what this is used for. While it is a Z-score, that's not really telling a user why its there / where its used. `cost_change_z_score` or `path_difference_z_score` or something would help give some context where/how its used. 

Considering the use of `consider_cost_change` in the action field, I think `cost_change_z_score` or `path_cost_change_score` or something would make sense

Reply:
Also parameter guide stuff :-)